RestaurantOrderingSystem/src/main/java/utils/DBManager.java

```diff
@@ -49,4 +50,20 @@ public static User getUser(String username) {
 		
 		return user;
 	}
+	
+	public static Order getOrder() {
```

Comment:
so as this is written this will grab the first order in the db and return it.  not a problem as long as we remove/fix it later, but we should probably have a TODO here